### PR TITLE
feat: add Deeper Network quickstart

### PR DESCRIPTION
**Pull request by Kav91**
# Summary

Adds a quick start for the Deeper Network's "connect" devices.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

deeperconnect01
deeperconnect02

### CODE REVIEW

File: quickstarts/deeper-network/config.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+authors:
+  - New Relic
+  - Kav. Pather
+
+documentation:
+  - name: Deeper Network
+    description: |
+      The Decentralized Gateway and Infrastructure for Web3.0. Deeper Network combines blockchain, network security, and sharing economy to create a global peer to peer network that empowers the real users of the internet and paves the way for the next generation of the web.
+    url: https://deeper.network/
```
Comment (polfliet) [inline quickstarts/deeper-network/config.yml:30]: Is there a specific docs page or GH readme that explains the steps for setting up Deeper Connect with New Relic?

**Review by polfliet — COMMENTED** [review]
I knew we could expect a contribution from you @Kav91! And we are looking forward to many more! Awesome!!
I just have one question if there is a specific doc page that explains how to set up Deeper network with New Relic? If that exists, we can add that as the documentation url.

File: quickstarts/deeper-network/dashboards/deeper-connect.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
     "name": "Deeper Network - Connect Monitoring",
     "description": null,
-    "permissions": "PUBLIC_READ_WRITE",
```
Comment (polfliet) [inline quickstarts/deeper-network/dashboards/deeper-connect.json]: @jpvajda @aswanson-nr this is another manual step that Kav had to do after exporting a dashboard from within NR1. How could we make this easier for contributors so they don't have to do this manually?

File: quickstarts/deeper-network/config.yml (hunk 1 above)
Comment (Kav91) [reply quickstarts/deeper-network/config.yml:30]: I intentionally did not add something specific right now, was just trying to coordinate when & where I make the guide & CLI utility public, if potentially under their website/github. 
But for better coverage right now I've added another documentation link and install plan, I will make it public when this PR is merged and/or the CLI utility made public. 

Hoping this should be sufficient for now from our end?

File: quickstarts/deeper-network/dashboards/deeper-connect.json (hunk 2 above)
Comment (Kav91) [reply quickstarts/deeper-network/dashboards/deeper-connect.json]: If it helps, whilst I was PR'ing the related [entity synthesis](https://github.com/newrelic/entity-definitions#dashboard-templates) for this, their repo contains a `sanitize dashboards` function that runs via docker compose. This handled everything for me, would be nice to have similar.

**Comment by Kav91** [comment]
@polfliet have added another documentation link and install plan. Hope this covers everything now 🤞🏾 

Found the overall process pretty straight forward, +having so many examples helped even further 👍🏾 👍🏾 👍🏾

**Comment by jpvajda** [comment]
@Kav91 Thanks for sharing! I highly respect your point of view on this, you are one of the most active contributors to our open source projects, 💯   so just let us know if you see any other areas we can improve!

File: quickstarts/deeper-network/dashboards/deeper-connect.json (hunk 2 above)
Comment (aswanson-nr) [reply quickstarts/deeper-network/dashboards/deeper-connect.json]: @Kav91 Great suggestion! I'll dig into that and get a ticket created for us.

Comment (jpvajda) [reply quickstarts/deeper-network/dashboards/deeper-connect.json]: yes, that was what @thezackm also mentioned we could use

Comment (jpvajda) [reply quickstarts/deeper-network/dashboards/deeper-connect.json]: @aswanson-nr I found this old issue which seem to be about this problem

https://github.com/newrelic/newrelic-quickstarts/issues/55

**Review by polfliet — APPROVED** [review]

**Comment by polfliet** [comment]
Nice, thanks @Kav91!

**Comment by jpvajda** [comment]
https://github.com/newrelic/newrelic-quickstarts/issues/425

**Pull request merged**